### ISSUE TITLE
Headers with Period Become a Body

### ISSUE THREAD

**Issue by alc0der** [Leopard2A5/fhttp#53]
Adding a dot "." inside a header name seems to convert the header to a body.

Sample:
```fhttp
# example.http
GET https://api.example.com
com.example: name@example.com
```

Command:
```sh
fhttp --curl example.http
```
> I have added `--curl` as a mean to debug

Output
```
curl -X GET \
-d "com.example: name@example.com\
--url https://api.example.com
```

**Comment by Leopard2A5** [comment]
Indeed! Thank you for the report!

**Comment by Leopard2A5** [comment]
fixed and released in 1.5.2
